### DISCUSSION TITLE
disable the magnet effect by default

### DISCUSSION THREAD

**Discussion by iberck** [pubpub-zz/ppInk#231] [Q&A]
Is there a way to disable the magnet effect (bar magnet icon) by default (opening pplnk) ?

**Comment by pubpub-zz** [comment]
Magnet can be disable setting the radius to 0 or a negative value:
![image](https://user-images.githubusercontent.com/4083478/206266574-61516eea-9d25-4d86-b440-861213d685ed.png)
But there was a regression and you were no more able to set it.
This version fixes this
[ppInk Disable Magnet.zip](https://github.com/pubpub-zz/ppInk/files/10178991/ppInk.Disable.Magnet.zip)

Thanks for report
